.changes/next-release/bugfix-400db503-7828-4789-8f61-8bf7d200fe08.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix prevents uncaught error on being thrown when downloading objects from deleted buckets (#1601)"
```

Comment:
"notifies the user of bucket unavailability when trying to open/download objects from deleted buckets. " from the description is a better user-centric description. How about something like "Fix being able to open/download objects from deleted S3 buckets (#1601)"